contrib/test_jenkins.sh

```diff
@@ -704,6 +717,12 @@ run_ucx_perftest() {
 		then
 			# Run UCP performance test
 			$MPIRUN -np 2 -x UCX_NET_DEVICES=$dev -x UCX_TLS=$tls $AFFINITY $ucx_perftest $ucp_test_args
+
+			# Run UCP performance test with 2 threads
+			$MPIRUN -np 2 -x UCX_NET_DEVICES=$dev -x UCX_TLS=$tls $AFFINITY $ucx_perftest $ucp_test_args -T 2
```

Comment:
can remove this one - it is already tested in client/server mode, and does not use loopback anyway